cli/cli.go

```diff
@@ -1312,3 +1313,37 @@ func (c *cmd) printServersForAutoComplete(country string, hasGroupFlag bool, gro
 		}
 	}
 }
+
+func parseConnectArgs(ctx *cli.Context) (string, string, error) {
+	args := ctx.Args()
+
+	// generate server tag from given args
+	var serverTag string
+	var serverGroup string
+	if args.Len() > 0 {
```

Comment:
[small] you can minimize nesting by inverting the condition here and returning early